src/components/organisms/batch-jobs-activity-list/index.tsx

```diff
@@ -134,12 +134,16 @@ const BatchJobActivityCard = ({ batchJob }: { batchJob: BatchJob }) => {
         <FileIcon fill={iconColor} size={20} />
```

Comment:
```suggestion
        const iconColor = clsx({ "text-grey-40": Math.abs(relativeTimeElapsed.raw) > twentyfourHoursInMs })
       
       // ...
       
        <FileIcon className={iconColor} size={20} />
```

nit: We should avoid hardcoding color values, as it makes it more difficult in the case of updates to our design system.